api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -601,15 +615,9 @@ public ResponseEntity<CloneWorkspaceResponse> cloneWorkspace(
 
     DbUser user = userProvider.get();
     // Note: please keep any initialization logic here in sync with createWorkspaceImpl().
-    String billingProject = createTerraBillingProject(accessTier);
```

Comment:
Moved the logic to the service to match the creation logic in the create case. Not necessary for the current change but I wanted the code to be consistent